macOS/DuckDuckGo/Menus/MainMenu.swift

```diff
@@ -451,6 +451,7 @@ final class MainMenu: NSMenu {
     func buildDebugMenu(featureFlagger: FeatureFlagger, historyCoordinator: HistoryCoordinating) -> NSMenuItem? {
 #if DEBUG || REVIEW || ALPHA
         NSMenuItem(title: "Debug")
+            .withAccessibilityIdentifier("Debug")
```

Comment:
How about we use a more specific identifier?
```suggestion
            .withAccessibilityIdentifier(“MainMenu.Debug")
```